画像サイズをデフォルトbigとする

s-nakajima opened the pull request.

s-nakajima committed:
画像サイズをデフォルトbigとする

otokomae commented:
この対応だと「選択して下さい」の値が "big" になってしまうので
プルダウンで「大」を選択しても値が変わらなくなってしまいました。

もし、big をデフォルト選択でよいのであれば
「選択して下さい」を取り払って、 大・中・小・サムネール　のみにして
初期選択値を「大」にするのはありでしょうか？

RikaFujiwara committed:
Wysiwygビヘイビア動作不正対応

１回のSaveの動作時に複数フィールドがWysiwyg処理対象フィールドであったり、
複数のモデルが１トランザクション中でsaveを行うとNotice,Warningが発生していたことへの対処

RikaFujiwara committed:
chmod

s-nakajima commented:
なるほど。では、「選択してください」をやめて、「大」を初期設定にするので構いませんが、ラベルを「大」ではなく「標準」に変えられませんか。

s-nakajima committed:
Merge pull request #78 from NetCommons3/behavior_fix

Behavior fix
バグで、各プラグインに影響あるのでマージします。

RikaFujiwara committed:
Wysiwygの内容のZIP化、ZIPからのWysiwygデータへの展開機能の追加

RikaFujiwara committed:
chmod, chown

s-nakajima committed:
Merge pull request #79 from NetCommons3/wysiwyg_zip

Wysiwygフィールドに設定されているデータをZIPファイルにまとめる機能の追加

s-nakajima committed:
Merge origin/master

s-nakajima committed:
文言を「標準サイズ」に変更
